rpm/skopeo.spec

```diff
@@ -7,15 +7,6 @@
 %global debug_package %{nil}
 %endif
 
-# RHEL's default %%gobuild macro doesn't account for the BUILDTAGS variable, so we
-# set it separately here and do not depend on RHEL's go-[s]rpm-macros package
-# until that's fixed.
-# c9s bz: https://bugzilla.redhat.com/show_bug.cgi?id=2227328
-# c8s bz: https://bugzilla.redhat.com/show_bug.cgi?id=2227331
-%if %{defined rhel}
-%define gobuild(o:) go build -buildmode pie -compiler gc -tags="rpm_crashtraceback libtrust_openssl ${BUILDTAGS:-}" -ldflags "-linkmode=external -compressdwarf=false ${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v -x %{?**};
```

Comment:
`libtrust_openssl` is not a global RHEL tag, it is specific to our `github.com/containers/libtrust` . Should we really just _drop it_?

I see that the c9s builds still include the tag (apparently it was hard-coded in `go-srpm-macros` in some versions/patches?) but the c10s builds don’t.

Should we add that tag back into our `BUILDTAGS` in `skopeo.spec`? Or is it truly OK to drop it?

Reply:
good catch. You're right and my bad for missing that. I'll fix that in spec in this PR itself. Putting back on draft.

Reply:
Alright, the latest build does show `libtrust_openssl` tag for centos stream builds: https://copr.fedorainfracloud.org/coprs/packit/containers-skopeo-2501/build/8566716/ . It will be set for rhel too.

I also included another commit to silence rpmlint.